build/publish.proj

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
+
+  <!--
+    ============================================================
+    This is for orchestrated build scenarios. The NuGet.Client-Orchestrated build definition builds this file directly.
+    ============================================================
+  -->
+
+  <Import Project="$(BlobPublishTargetsPath)" />
+  <Import Project="$(BuildCommonDirectory)common.targets" />
+
+  <ItemGroup>
+    <ItemsToPush Include="$(NupkgOutputDirectory)*.nupkg" />
+  </ItemGroup>
+
+  <Target Name="Build" Condition=" '$(BuildRTM)' == 'false' AND Exists($(NupkgOutputDirectory))">
```

Comment:
BuildRTM == VS insertion?

Reply:
No this is a configuration variable in our build definition that we use to condition steps in our pipeline. This allows us to have 2 parallel builds which can do different steps.